packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -384,6 +393,11 @@ export abstract class FluidDataStoreContext
 						value: this.id,
 						tag: TelemetryDataTag.CodeArtifact,
 					},
+					packageName: {
+						value: JSON.stringify(this.pkg),
+						tag: TelemetryDataTag.CodeArtifact,
+					},
+					stack: generateStack(),
```

Comment:
Do we need to collect stack here? I.e. errors always collect stack. It might be coming from original error, but I think it's even better.

Reply:
This is what the current logs look like. I am not actually sure if explicitly adding a stack is better than this or not.

<HTML><HEAD><TITLE>From Clipboard</TITLE></HEAD><BODY><!--StartFragment-->

Data_eventName | Data_stack
-- | --
fluid:telemetry:RealizeError | Error    at We.rejectDeferredRealize (https://dev.loop.microsoft.com/fluid/stg/loop-app/hashed/99836.2f628a764ba6b5b18f79.js:2:72477)    at We.factoryFromPackagePath (https://dev.loop.microsoft.com/fluid/stg/loop-app/hashed/99836.2f628a764ba6b5b18f79.js:2:73205)    at async We.realizeCore (https://dev.loop.microsoft.com/fluid/stg/loop-app/hashed/99836.2f628a764ba6b5b18f79.js:2:73676)
fluid:telemetry:Summarizer:Running:Summarize_cancel | Error    at We.rejectDeferredRealize (https://dev.loop.microsoft.com/fluid/stg/loop-app/hashed/99836.2f628a764ba6b5b18f79.js:2:72477)    at We.factoryFromPackagePath (https://dev.loop.microsoft.com/fluid/stg/loop-app/hashed/99836.2f628a764ba6b5b18f79.js:2:73205)    at async We.realizeCore (https://dev.loop.microsoft.com/fluid/stg/loop-app/hashed/99836.2f628a764ba6b5b18f79.js:2:73676)


<!--EndFragment--></BODY></HTML>

Reply:
It seems adding this stack will be a no-op since the stack from the error will be used. I will remove it.